spec/models/contact_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+describe Contact, type: :model do
+  it { is_expected.to belong_to(:organization).required }
+  it { is_expected.to validate_presence_of(:organization) }
+  it { is_expected.to validate_presence_of :contact_type }
+  it { is_expected.to validate_presence_of :data }
+
+  context 'when contact_type present' do
+    let(:contact) { create(:contact) }
+
+    it 'validate if valid' do
+      expect(contact).to be_valid
+    end
+
+    it 'validate if invalid' do
+      contact.data = ''
+      expect(contact).not_to be_valid
+    end
+  end
+end
```

Comment:
Should we add specs for your validations of contact types?

Reply:
They are not different at the moment, it is better to add specs when additional functional will be added